snapcraft/cli/lifecycle.py

```diff
@@ -272,7 +298,8 @@ def pack(directory, output, **kwargs):
     required=False,
     help="Forces snapcraft to use LXD for this clean command.",
 )
-def clean(parts, use_lxd):
+@click.option("--unprime", is_flag=True, required=False, cls=HiddenOption)
```

Comment:
Just a note, newer click versions added hidden=true so we can use that when we upgrade click.